dash-renderer/tslint.json

```diff
@@ -1,7 +1,8 @@
 {
     "defaultSeverity": "error",
     "extends": [
-        "tslint:recommended"
+        "tslint:recommended",
+        "tslint-config-prettier"
```

Comment:
I think "quotemark" should either be removed or updated to:
        "quotemark": [true, "single", "avoid-escape"],

So that tslint doesn't disagree with prettier if wrapping a " ' ' ' " or ' " " " ' depending on settings.

Reply:
yeah we should probably remove a whole bunch of these overrides... I just got it far enough that they didn't fight with each other in the configuration I saw. For now I'll just 🔪 `quotemark`.